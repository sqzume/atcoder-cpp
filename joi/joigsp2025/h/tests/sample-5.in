15 10000000 1000000000000
160278118759 43084 33592
442653603914 19490 23090
824219815410 50858 89563
502303340628 56629 45080
495062829942 87342 28821
234536700105 45384 34328
396080693809 78081 50812
734374391045 40873 92012
122606844331 25451 30426
204076581972 58431 13989
495156368673 54276 41670
812963939390 27614 50228
405067019838 96324 18477
464546304875 67562 45956
528559327980 41759 15546
10
216000000000000
1728000000000000
5832000000000000
13824000000000000
27000000000000000
46656000000000000
74088000000000000
110592000000000000
157464000000000000
216000000000000000
