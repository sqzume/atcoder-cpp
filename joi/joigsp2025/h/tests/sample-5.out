995176
1135557
1431775
1824183
2359362
3059523
3942014
5106209
6594716
8448125
